Views/sessions.html

```diff
@@ -0,0 +1,140 @@
+            });
+        });
+    });
+     // Script to POST URL encoded data from a HTML form
+     $(document).ready(function(){
+         // click on button submit
+         $("#submitPostRaw").on('click', function(){
+             // send ajax
+             $.ajax({
+                 url: '/rawsession', // url where to submit the request
+                 type : "POST", // type of action POST || GET
+                 contentType: "application/x-www-form-urlencoded", // set the request Content-Type header
+                 dataType : 'json', // data type
+                 data : $("#post_form_raw").serialize(), // post data || get data
+                 success: function(data, status){
+                     alert("Server response data: " + JSON.stringify(data) + "\nStatus: " + status);
+                 },
+                 error: function() {
+                     alert("Failed to post JSON");
+                 }
+             })
+         });
+     });
+     // Script to send a DELETE request to '/sessions'
+    $(document).ready(function(){
+        $('#deleteButtonRaw').click(function(){
+            $.ajax({
+              url: '/rawsession',
+              type: 'DELETE',
+              success: function(data,status){
+                  alert("Server successfully deleted \nStatus: " + status);
+              }
+            });
+        });
+    });
+</script>
+   <title>Sessions</title>
+ </head>
+ <body>
+     <p>
+     This page will send HTTP requests to the "/session" and "/rawsession" routes of the Kitura sample server.<br>
+     A session is created for each user and is persisted through cookies. You can test this using a private window.<br>
+     The server will process the incoming request to return or change an array "Books" that is stored within the user's session.<br>
+     <a href="https://github.com/IBM-Swift/Kitura-Sample/blob/master/Sources/Application/Routes/SessionsRoutes.swift" target="_blank">View the code</a>
+     </p>
+    <ul><li>
+         <p>Send an HTTP GET request to &quot;/session&quot; to see the books in your session</p>
+         <input type="button" value="GET to &quot;/session&quot;" id="getButton" >
+     </li></ul>
+     <ul><li>
+         <p>Send a URL encoded HTTP POST request to &quot;/session&quot; to add a book to your session</p>
+         <form action="" method="post" id="post_form">
+             Name: <input type="text" name="name" required="required"><br>
+             author: <input type="text" name="author" required="required"><br>
+             rating: <input type="number" name="rating" required="required" min="1" max="5"><br>
+             <input id="submitPost" type="button" name="Submit" value="Submit">
+         </form>
+     </li></ul>
+     <ul><li>
+         <p>Send an HTTP DELETE request to &quot;/session&quot; logout and clear your session data</p>
+         <input type="button" value="DELETE to &quot;/session&quot;" id="deleteButton">
+     </li></ul>
+     <ul><li>
```

Comment:
Maybe add headers to show the difference between the codable options and the raw ones, the button text is easy to overlook